src/models/data.py

```diff
@@ -485,7 +500,12 @@ def calculate_static_normalizing_array(
             for norm_var in normalizing_dict_keys:
                 if var == norm_var:
                     mean.append(self.static_normalizing_dict[norm_var]["mean"])
-                    std.append(self.static_normalizing_dict[norm_var]["std"])
+                    # TODO: DO NOT ALLOW STD TO BE ZERO
```

Comment:
Is this the correct way to enforce htat std is not 0? Should be be something really small if the variation is low rather than 1.0? .e.g `1e-10` ?

Reply:
If the variance is equal to 0, dividing by a very small number won't change the variance (especially because the values will be equal to 0 when we subtract the mean), so its easier to just divide by 1 and leave it as is.